btree/btree.c

```diff
@@ -2039,15 +2037,15 @@ static int64_t node_get(struct node_op *op, struct td *tree,
 		 * segment. Take up with BE segment task.
 		 */
 		M0_ASSERT(node != NULL);
-		node->n_addr         = *addr;
-		node->n_tree         = tree;
-		node->n_type         = nt;
-		node->n_seq          = m0_time_now();
-		node->n_ref          = 1;
-		node->n_txref        = 0;
-		node->n_delayed_free = false;
+		node->n_addr          = *addr;
+		node->n_tree          = tree;
+		node->n_type          = nt;
+		node->n_seq           = m0_time_now();
+		node->n_ref           = 1;
+		node->n_txref         = 0;
+		node->n_be_node_valid = false;
```

Comment:
Well, the variable name does not match with the use. If we have allocated the node then it means node is valid but here it is set to false.

Reply:
Right, made changes according to new name.